### DISCUSSION TITLE
Temporarily hide read posts

### DISCUSSION THREAD

**Discussion by Maxplosion** [Docile-Alligator/Infinity-For-Reddit#974] [Completed Ideas] [answered]
Hi! 

A feature request to add a button to temporarily hide read posts for the current session. 

Example: 
10 new posts on the sub (or front page), I read 5 of them, so 5 unread, 5 read, press button, only the 5 unread are left and shown, rest is hidden. 
If I now manually reload the subreddit page (or open and close the app) all 10 posts, where 5 are marked as read and 5 are marked as unread are shown again. 

In the current implementation with hiding read posts using the settings switch makes it so if I want to reread a read post I have to use the toggle in the settings, search the post, read the post and activate it again in the settings - a rather cumbersome workflow.

Thank you!

**Comment by chesamo** [comment]
To add to this, some way to visually distinguish betwee  read and unread posts would help a lot. (Perhaps by faiding the contrast of the post title?)

**Reply by Maxplosion** [reply]
that is already possible to do - go to Settings > Post History > Mark Posts as Read.

Depending on your theme this then changes the read posts (in most cases it makes them, like you suggested, faded)

**Reply by chesamo** [reply]
This should be the default behavior.

**Comment by Maxplosion** [comment, marked as answer]
I just found that this has been added by longpressing the '+' button at the bottom right of the page. Exactly what I wanted, thx!

**Reply by Maxplosion** [reply]
@Wladefant you can close this idea if you want - (I have no clue how to mark anything as solved in here...)

**Reply by Wladefant** [reply]
Unfortunately not possible to mark it as solved (why github), just change the category to completed ideas 😄
